lib/slack/src/index.ts

```diff
@@ -15,4 +12,40 @@ app.get('/ping', async (_req, res) => {
 
 })
 
+// This is designed as a Rest endpoint, although the HR api is GraphQL. I'm not sure which
+// is best for this, but presumably this slack api is a Backends for Frontends type Api, in
+// which case Rest, or Rpc is probably going to be more suitable that GraphQl.
+app.get('/:userId/manager', async (req, res) => {
+  // We will probably want to use relay or something later as the GraphQl usage gets more 
+  // complex, but I think hand rolling things is fine for now
+  fetch(`http://localhost:${daprPort}/v1.0/invoke/hr.hr/method/graphql`, {
+    "body": JSON.stringify(
+      {
+        "query": managerQuery,
+        "variables": { userId: req.params.userId }
+      }),
+    "method": "POST",
+  })
+    .then(response => response.json())
+    .then(json => res.send("Respose from hr api: " + json.data.human.manager.name))
+    .catch(error => console.log(error))
```

Comment:
I'm fine with hand-rolled graphql for one-offs. To me, the main advantage of graphql for interfacing between different languages is that there is excellent tooling for generating typescript types from the graphql schema (whereas `response.json()` returns `any`).

I would have a slight preference for writing this as an async function. Partly for consistency with the function above, and partly because it's likely to get more complex as it gets turned into a real slack command handler.

I think you're supposed to pass errors back to express using `next()` - https://expressjs.com/en/guide/error-handling.html . Honestly, express is one of my least favourite web server frameworks though, and maybe it does the right thing (500) in this case anyway? In which case it seems fine to leave it.

Reply:
I agree, strong typing the response is a good next step, I'll add it to the backlog

Reply:
I've updated to use await and to adhere to express error handling best practices (I think)